### PR TITLE
[Device Introduction article] Update generic sensor link

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Point to the new article now that https://github.com/GoogleChrome/web.dev/pull/4603 has landed.

### CODE REVIEW

File: src/site/content/en/blog/devices-introduction/index.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ Photo by [Darya Tryfanava] on [Unsplash].
 [Build your first WebAuthn app]: https://codelabs.developers.google.com/codelabs/webauthn-reauth/
 [File System Access API]: /file-system-access/
 [File API]: https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications
-[Generic Sensor API]: https://developers.google.com/web/updates/2017/09/sensors-for-the-web
+[Generic Sensor API]: https://web.dev/generic-sensor/
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/devices-introduction/index.md:191]:
```suggestion
[Generic Sensor API]: /generic-sensor/
```

**Review by beaufortfrancois — APPROVED** [review]
LGTM with nit

**Review by mihajlija — APPROVED** [review]

**Comment by kaycebasques** [comment]
@tomayac merge yourself when ready please (Francois mentioned a nit and I'm not sure if that has been resolved)

**Comment by tomayac** [comment]
> @tomayac merge yourself when ready please (Francois mentioned a nit and I'm not sure if that has been resolved)

Merged. The nit was that I had initially linked to https://web.dev/generic-sensor/ instead of just /generic-sensor/ as I did now.

**Pull request merged**